packages/regl-worldview/src/commands/FilledPolygons.js

```diff
@@ -44,11 +46,17 @@ function getEarcutPoints(points: Vec3[]): Vec3[] {
 
 type Props = {
   children: PolygonType[],
-  getHitmapId?: (TriangleList) => number,
```

Comment:
I think this is a backwards incompatible change - is it a lot of work to still support this prop & pass it through to the `<Triangles />` command?

Reply:
Not a lot of work. Let me add it